sdk/storage/Azure.Storage.Queues/api/Azure.Storage.Queues.netstandard2.0.cs

```diff
@@ -1,5 +1,12 @@
 namespace Azure.Storage.Queues
 {
+    public partial class InvalidMessageEventArgs : Azure.SyncAsyncEventArgs
```

Comment:
InvalidQueueMessageEventArgs ? To make it a little bit more specific.

Reply:
That was original idea. Here's the struggle:
It can also have `PeekedMessage` not only `QueueMessage` so I was avoiding to be specific.
Maybe we should have two properties or even two events instead?

as for collapsing this to single `object` property - we somewhat follow this https://github.com/Azure/azure-sdk-for-net/blob/f7d760cfdf1fb751d1f9ad540387c45b7086e4e9/sdk/storage/Azure.Storage.Queues/src/QueueClientSideEncryptionOptions.cs#L31-L34

@tg-msft what do you think?

Reply:
I would change the name to `InvalidQueueMessageEventArgs` because we have so many things called `Message`.  I really like the idea of having two properties for `ReceivedMessage` and `PeekedMessage` where only one will have a value.  That was too annoying when we're passing things to a `Func<...>`, but there's plenty of room for more specific, helpful named values in an EventArgs class.

Reply:
Okay. The type has been renamed to `QueueMessageDecodingFailedEventArgs` with `ReceivedMessage` and `PeekedMessage` props. I went with `Queue` property (I like it more and kind of used to such naming in webjobs extension, i.e. `BlobClient blob; QueueClient queue` etc.)